cli/src/commands/init.rs

```diff
@@ -53,13 +55,21 @@ pub(crate) fn cmd_init(
         .and_then(|_| wc_path.canonicalize())
         .map_err(|e| user_error_with_message("Failed to create workspace", e))?;
 
+    // Preserve existing behaviour where `jj init` is not able to create
+    // a collocated repo.
+    let collocated = false;
```

Comment:
nit: I don't know if there's a difference between "colocated" and "collocated", but we seem to consistently spell it with one "l" everywhere else, so let's do that here too.

Reply:
My bad. That's a typo on my part. Fixed.